test/constants.ts

```diff
@@ -39,3 +39,7 @@ export const zeroBytes32 = "0x00000000000000000000000000000000000000000000000000
 export const identifier = utf8ToHex("IS_ACROSS_V2_RELAY_VALID");
 
 export const zeroRawValue = { rawValue: "0" };
+
+export const spokePoolRelayerRefundRoot = createRandomBytes32();
```

Comment:
I have a const for this in my other PR called `mockDestinationDistributionRoot`.

Reply:
Will merge that in